### PR TITLE
Fe feat/proptypes

### PR DESCRIPTION
**Pull request by jmartin777**
# Description
Implemented Prop-Types into 

-App
-Details Page
-Field Page
-Mushroom Card
-Toggle Save
-Upload Image

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

-Prop-Types

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [ ] Refactoring

# How Has This Been Tested?

Only through App user interaction.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Elise-Jones — APPROVED** [review]

File: src/Components/App/App.js [hunk 1]
```diff
@@ -76,4 +77,10 @@ function App() {
   );
 }
 
+App.propTypes = {
```
Comment (andreasorensen) [inline src/Components/App/App.js]: App doesn't take in any props, so I don't think we need this...

Comment (Elise-Jones) [reply src/Components/App/App.js]: good catch no prop types for app

**Review by andreasorensen — CHANGES_REQUESTED** [review]
Thanks for implementing prop types, always good to have that check. The one thing is, the App component doesn't take in an props, so it doesn't need any proptypes.

File: src/Components/App/App.js (hunk 1 above)
Comment (jmartin777) [reply src/Components/App/App.js]: sounds good making the change now and pushing up

**Pull request merged**